Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
```

Comment:
Use a smaller base image: Instead of using the python:3.10 image, you can use a smaller base image like python:3.10-slim or python:3.10-alpine. These images are smaller in size and will result in smaller image size for your application.